backend/database.py

```diff
@@ -1,13 +1,45 @@
-from peewee import *
 import json
+import sys
 
-config = json.load(open("./backend/config.json", "r"))
+from sqlalchemy import create_engine, exc
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
 
-db = PostgresqlDatabase(None)
 
-db_name = config["DATABASE"]
-db_host = config["PGHOST"]
-db_user = config["USER"]
-db_password = config["PASSWORD"]
+def init_db():
+    config = json.load(open("./backend/config.json", "r"))
 
-db.init(db_name, host=db_host, user=db_user, password=db_password)
+    db_name = config["DATABASE"]
+    db_host = config["PGHOST"]
+    db_port = config["PORT"]
+    db_user = config["USER"]
+    db_password = config["PASSWORD"]
+
+    SQLALCHEMY_DATABASE_URL = (
+        f"postgresql://{db_user}:{db_password}@{db_host}:{db_port}/{db_name}"
+    )
+    engine = create_engine(SQLALCHEMY_DATABASE_URL)
+
+    try:
+        connection = engine.connect()
+        print("🔗 Database connection established")

```

Comment:
we need to open a task to migrate to using `logger` instead of prints

Reply:
opened